src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -157,6 +187,26 @@ public void resetFlywheelPosition() {
     shooterLeft.getSensorCollection().setIntegratedSensorPosition(0.0, 0);
   }
 
+  public void setTargetNearRpm(double target) {
+    targetNearRpm = target;
+  }
+
+  public void setRpmTargetUsingDistance(double distance) {
+    if (distance < 8.5) {
+      retractHood();
+    }
+    if (distance > 9) {
```

Comment:
- [ ] Replace 8.5 and 9 with constants: HOOD_CHANGE_DISTANCE = 8.75 and HOOD_CHANGE_DISTANCE_THRESHOLD = .25